docs/content/en/4.providers/twicpics.md

```diff
@@ -5,16 +5,437 @@ navigation:
+
+### turn
+
+`turn` will change the orientation of the image. It accepts an angle in degrees or an expression. Angles will be rounded to the closest multiple of 90°.
+
+Syntax: `{ turn: `[`number`](https://www.twicpics.com/docs/api/manipulations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#number)`|'flip'|'left'|'right' }`
+
+```html
+<NuxtImg
+  provider="twicpics"
+  src="/football.jpg"
+  :modifiers="{turn:'left'}" <!-- turns image at -90° -->
+/>
+
+<NuxtImg
+  provider="twicpics"
+  src="/football.jpg"
+  :modifiers="{turn:180}" <!-- turns image at 180° -->
+/>
+```
+
+More informations [about `turn` here](https://www.twicpics.com/docs/api/transformations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#turn).
+
+### zoom
+
+Zooms into the image by a factor equal or superior to 1 towards the focus point while preserving the image size.
+
+Syntax: `{ zoom: `[`number`](https://www.twicpics.com/docs/api/manipulations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#number)` }`
+
+```html
+<NuxtImg
+  provider="twicpics"
+  src="/components/flip/cherry-3.jpg"
+  :modifiers="{zoom:1.5}" <!-- zooms into image by a factor 1.5 -->
+/>
+
+<NuxtImg
+  provider="twicpics"
+  src="/components/flip/cherry-3.jpg"
+ :modifiers="{zoom:3}" <!-- zooms into image by a factor 3 -->
+/>
+
+<NuxtImg
+  provider="twicpics"
+  src="/components/flip/cherry-3.jpg"
+ :modifiers="{focus:'auto', zoom:3}" <!-- zooms into image by a factor 3 in direction of the focus-->
+/>
+
+<NuxtImg
+  provider="twicpics"
+  src="/components/flip/cherry-3.jpg"
+ :modifiers="{focus:'200x200', zoom:3}" <!-- zooms into image by a factor 3 in direction of the focus-->
+/>
+
+```
+
+More informations [about `zoom` here](https://www.twicpics.com/docs/api/transformations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider#zoom).
+
+## Combination of parameters 
+
+You can combine several transformations of the [TwicPics API](https://www.twicpics.com/docs/api/transformations/?utm_source=nuxt&utm_medium=organic&utm_campaign=provider).
+
+Be aware that the order of the parameters can be significant.
+
+Example:
+
+This will return a variant of image for which we have, in order:
+1. cropped the image from the center to 16:9 aspect ratio
+2. then placed the focus on the center of interest of the cropped image
+3. then rotate the image 90° to the left
+
+The result is a 9:16 (not 16:9) image with a possibly false area of interest.
+```html
+<nuxt-img
+  provider="twicpics"
+  src="/football.jpg"
+  :modifiers="{cover:'16:9', focus:'auto', turn:'left'}"
+/>
+```
+
+This will return a variant of your image for which we have, in order:
+1. placed the focus on the center of interest of the original image
+2. then cropped the image to 16:9 from the center of interest
+3. then rotated the image 90° to the left
+
+The result is a cropped image with the area of interest retained and displayed in 16:9 format.
+```html
+<nuxt-img
```

Comment:
```suggestion
<NuxtImg
```

Reply:
> Added some comments, thank you for this PR ❤️

Thanks for this early review. :thumbsup: